packages/xdl/src/ip.js

```diff
@@ -44,6 +44,13 @@ function isPublic(addr) {
   return !isPrivate(addr);
 }
 
+function isVirtualNetworkAdapter(macAddr) {
+  var OUIVendors = [
+    '00:50:56' // VMware Network Adapter
+  ];
+  return OUIVendors.map( (oui) => {return macAddr.startsWith(oui)} )[0];
```

Comment:
This seems a bit more complicated than necessary, can you do just:
```js
const VMWARE_OUI = '00:50:56';
// ...
return macAddr.startsWith(VMWARE_OUI);
```

Or if the idea is to have a list of OUIs that can be later added to, then it should check each of them, like:
```js
const VIRTUAL_OUIS = [
  '00:50:56', // VMware
];
// ...
return VIRTUAL_OUIS.some(oui => macAddr.startsWith(oui));
```

Reply:
Definitely better, especially the second one. Thank you 👍